packages/fether-react/src/Tokens/TokensList/TokenAddress/TokenAddress.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import React, { Component } from 'react';
+import { inject } from 'mobx-react';
+import PropTypes from 'prop-types';
+import { AccountCard } from 'fether-ui';
+import { withRouter } from 'react-router-dom';
+
+import withAccount from '../../../utils/withAccount';
+
+@withRouter
+@withAccount
+@inject('sendStore')
+class TokenAddress extends Component {
+  static propTypes = {
+    copyAddress: PropTypes.bool,
+    shortAddress: PropTypes.bool,
+    token: PropTypes.object
+  };
+
+  handleClick = () => {
+    const {
+      account: { address },
+      history,
+      sendStore,
+      token
+    } = this.props;
+
+    if (!token.address) {
+      return;
+    }
+
+    sendStore.clear();
+    history.push(`/send/${token.address}/from/${address}`);
+  };
+
+  render () {
+    const {
+      account: { address, name, type },
+      copyAddress,
+      shortAddress
+    } = this.props;
+
+    return (
+      <AccountCard
+        address={address}
+        copyAddress={copyAddress}
+        name={name}
+        onClick={this.handleClick}
+        shortAddress={shortAddress}
+        type={type}
+        {...this.props}
```

Comment:
```suggestion
```

I think we can remove this line? Or else it'll pass stuff like sendStore, match and other props to AccountCard

Reply:
Thanks, looking into this change I noticed that there were props I was passing down with TokenAddress unnecessarily such as `tokens` and `onClick` (they were used in TokenBalance but not required in TokenAddress). I've updated it to explicitly pass down `drawers` prop, as it's needed by the Card child component (otherwise it doesn't render the anything on the page other than who it's from). I've pushed commits that resolve the issues.

I also noticed that the SignedTxSummary (confirmation page after sending a tx from a Parity Signer account) didn't have the 'To' and 'Amount' in the right order, and the 'Amount' field wasn't styled to look consistent to how it looks on the Unlock page (confirmation page after sending a tx from a non-Parity Signer account), so I fixed that too as shown below

<img width="361" alt="screen shot 2019-01-24 at 1 24 25 pm" src="https://user-images.githubusercontent.com/6226175/51650581-ee224400-1fdc-11e9-9b6c-690790c7f3a7.png">

I also had to change the logic in the Name component, as it's actually only on the "Account" page where we need extra top/bottom padding and need to use classes `account_name -header`, but on the "Accounts List" page and the confirmation pages we just want the Name component to use `account_name`